qiskit_dynamics/backend/backend_utils.py

```diff
@@ -173,44 +171,43 @@ def _get_counts_from_samples(samples: list) -> Dict:
 
 
 def _iq_data(
```

Comment:
pretty minor but can you change this name to `_get_iq_data` to be consistent with the other data generating functions in this file